oml/functional/metrics.py

```diff
@@ -209,6 +218,55 @@ def calculate_accuracy_on_triplets(embeddings: torch.Tensor, reduce_mean: bool =
         return acc
 
 
+def calc_fnmr_at_fmr(pos_dist: torch.Tensor, neg_dist: torch.Tensor, fmr_vals: Tuple[int, ...] = (1,)) -> torch.Tensor:
+    """
+    Function to compute False Non Match Rate (FNMR) value when False Match Rate (FMR) value
+    is equal to ``fmr_val``.
+
+    Args:
+        pos_dist: distances between samples from the same class
+        neg_dist: distances between samples from different classes
+        fmr_vals: Values of ``fmr`` (measured in percents) for which we compute the corresponding ``FNMR``.
+                  For example, if ``fmr_values`` is (20, 40) we will calculate ``FNMR@FMR=20`` and ``FNMR@FMR=40``
+
+    See:
+        https://en.wikipedia.org/wiki/Biometrics#Performance
```

Comment:
please, format links in this way: https://github.com/OML-Team/open-metric-learning/blob/7e8406daab3b48946c81d12b5ed85c7b7f91351b/oml/samplers/balance.py#L17

it's needed if we want to add this function to the documentation (I guess we gonna do it later, with other similar functions after the refactoring that @dapladoc proposed before)